mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -190,6 +192,146 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class MultiViewWrapper():
+    """Wrap transformation from single-view into multi-view.
+
```

Comment:
May explain how it works, for example, does it apply similar types of transformation with different random parameters? Does it works with keys other than img?